### DISCUSSION TITLE
Decentralized Exchange Feature

### DISCUSSION THREAD

**Discussion by ghost** [deso-protocol/dips#72] [Medium CIPs]
You know, this was the year of the DEX. Dozens of successful or unsuccessful projects have been made. But traders and investors loved it. I think we need this feature for project independence. Also, I think the developer community has the capability to do it. After all, there will be millions of different coins on this platform. hundreds of transactions per second.

my thoughts; If we can bring decentralized exchange features (stake, farming) to the clout blockchain, the project will reach much more people. Clout's value will rise to utopian values.

**Comment by F041** [comment]
Why do you think this belongs to core devs and not community devs?
